src/containers/Home.js

```diff
@@ -41,10 +47,18 @@ const MODULES = [
 
 const PETITION_GROUPS = [
   {
+    group: 'closed',
+    query: {
+      state: 'closed',
+      limit: 3,
+      sort: 'created'
```

Comment:
Does this sort by 'created' descending? Maybe there is also a way to sort by answer date (descending) but I am not sure if this available through the API